packages/ui-components/src/components/Inputs/Autocomplete.js

```diff
@@ -68,6 +69,7 @@ export const Autocomplete = ({
     value={value}
     disabled={disabled}
     onChange={onChange}
+    groupBy={option => option[optionGroupKey]}
```

Comment:
Added 'groupBy' prop input. If 'optionGroupKey' is undefined, the drop down will not be grouped.